services/repository/src/main/java/org/sagebionetworks/auth/controller/AuthenticationController.java

```diff
@@ -335,4 +341,75 @@ public void unbindExternalAliasFromAccount(
 		authenticationService.unbindExternalID(userId, providerEnum, alias);
 	}
 
+
+	/**
+	 * Issues a personal access token to authorize scoped access to Synapse resources. To use the token, use the HTTP

```

Comment:
In the documentation of the authentication controller differentiate between access tokens and personal access tokens, including the facts that (1) PATs are not linked to any third party OAuth client and (2) while ATs have a strict 24 hour lifetime, PATs are long lived.  As such it's critical to treat PATs as sensitive credentials, like passwords.